python/rfnoc_modtool/rfnoc-newmod/cmake/Modules/Findfpga.cmake

```diff
@@ -3,13 +3,13 @@ PKG_CHECK_MODULES(PC_FPGA fpga)
 
 FIND_PATH(
     UHD_FPGA_DIR
-    NAMES /usrp3/top/Makefile.common
+    NAMES /usrp3_rfnoc/top/Makefile.common
```

Comment:
usrp3_rfnoc was intended to be temporal (and yes, it generated the need for this kind of changes that you are proposing). However, 'usrp3_rfnoc' will go away rather soon and this changes would have to be reverted if accepted.